README.md

```diff
@@ -42,19 +42,30 @@ The app is extremley easy to use (and self explantory):
 6. That's it :)
 
 The whole process is very simple and automated, so you don't need to worry about anything else.\
-Setting up should take less then a minute, and syncing should take about 1 second per photo (due to Google's API rate limitiations of 60 requests per user per minute)
+Setting up should take less then a minute, and syncing should take about 1 second per photo (due to Google's API rate limitations of 60 requests per user per minute)
 
 ## How to Run Locally
 
 In order for the backend to function, it requires an OAuth client id and secret + an API key.\
 Since (for obvious reasons) this is a private app, you will need to create one for your own.\
 You can see instructions on how to do that [here](https://developers.google.com/workspace/guides/create-credentials).\
-Once you do that, create the file `server/.env`, and set the following environment variables:
+You will need OAuth Client ID credentials, as well as API key credentials.
 
-- `GOOGLE_CLIENT_ID`
-- `GOOGLE_CLIENT_SECRET`
+Rename [`server/sample.env`](server/sample.env) to `server/.env` and fill in the values.
```

Comment:
It's still possible to use these as ENV vars, let's update the description so it's clear that you can either use the `.env` file or set the ENV vars.

Reply:
Done